src/parse/expr.rs

```diff
@@ -422,9 +433,9 @@ impl<I: Iterator<Item = Lexeme>> Parser<I> {
                     location: token.location,
                     lval: false,
                     expr: match token.data {
-                        Token::Star => ExprType::Mul(Box::new(p_left), Box::new(right)),
-                        Token::Divide => ExprType::Div(Box::new(p_left), Box::new(right)),
-                        Token::Mod => ExprType::Mod(Box::new(p_left), Box::new(right)),
+                        Token::Star => ExprType::Binary(BinaryOp::Mul,Box::new(p_left), Box::new(right)),
+                        Token::Divide => ExprType::Binary(BinaryOp::Div,Box::new(p_left), Box::new(right)),
+                        Token::Mod => ExprType::Binary(BinaryOp::Mod,Box::new(p_left), Box::new(right)),
```

Comment:
Are there supposed to be spaces after these commas?

Reply:
Filed https://github.com/rust-lang/rustfmt/issues/4135